src/utils/transform.ts

```diff
@@ -223,6 +239,13 @@ function getProcess(segment: XrayTraceDataSegment): [string, TraceKeyValuePair[]
   return [segment.Document.name, tags];
 }
 
+function getLogGroup(document: XrayTraceDataSegmentDocument) {
```

Comment:
Right now only getting lambda function log group is clear here. For some custom services, this will probably return 'unknown' still and the user will have to fill in the correct log group manually.